hax/treemakers/posrec.py

```diff
@@ -131,7 +130,9 @@ def extract_data(self, event):
         aft = self.aft_map.get_value(self.x[i], self.y[i], self.z[i])
         event_data['s1_area_fraction_top_probability_hax'] = binom_test(
             size_top, size_tot, aft)
-
+        event_data['s1_area_fraction_top_probability_hax2'] = binom_test_pax(
```

Comment:
@darrylmasson @coderdj what's the difference with this one? (better name than ```hax2```?)

Reply:
The binom_test from scipy is only valid for integer inputs of PE. The version I put into pax accepts floating points and is more accurate. I don't think there's a particularly good reason to include the scipy-based calculation (unless we want to compare with the original SR0 codes)

Reply:
Yuck. OK I had them both for testing. The scipy one fails miserably below ~10pe. The plan is to revert to one before merging. The one will be the pax one I guess.

Reply:
Fixed in https://github.com/XENON1T/hax/pull/180